web/src/components/CreateButton.tsx

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import styled from '@emotion/styled';
+import React from 'react';
+
+const StyledButton = styled.button`
+  margin-right: 16px;
+  padding: 8px 16px;
+  border: none;
+  background-color: #fcba19;
+  color: #003366;
+  text-transform: uppercase;
+  letter-spacing: 0.2em;
+  border-radius: 2px;
+  cursor: pointer;
+  -webkit-transition-duration: 0.4s; /* Safari */
+  transition-duration: 0.4s;
+`;
+
+interface ICreateButtonProps {
+  onClick: (e: any) => void;
+  buttonText: string;
+};
+
+const CreateButton: React.FC<ICreateButtonProps> = (props) => {
+  const { onClick, buttonText } = props;
```

Comment:
```js
const { children, ...rest } = props;

return <StyledButton {...rest}>{children}</StyledButton>
```

Reply:
<CreateButton style={{backgroundColor: formValid? 'blue': 'red'}}>save!</CreateButton>;

const SpecialCreateButton = styled(CreateButton)`
  box-shadow: '2px 2px 0 0';
`

Reply:
The create button inside nav bar doesn't seem to relate to the formValid state. We could review it if you like.